roottest/root/io/double32/CMakeLists.txt

```diff
@@ -1,7 +1,22 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOTTEST_COMPILE_MACRO(other.C
+                       FIXTURES_SETUP other-fixture)
+
+ROOTTEST_ADD_TEST(template32
+                  MACRO runtemplate32.C+
+                  OUTREF template32.ref
+                  OUTCNVCMD sed -e s,offset=[[:space:]][[:space:]]*[0-9]*,offset=__deleted__,g
```

Comment:
```suggestion
                  OUTCNVCMD sed -e s,offset=[[:space:]]*[0-9]*,offset=__deleted__,g
```

Reply:
Yes, I rebase code and applied exactly such patch. Plus modified ref file.